src/commands.py

```diff
@@ -13,19 +13,20 @@
 from src.tools.state_machines import State, Action, Machine
 from discord import ButtonStyle, Interaction
 from typing import List
+import asyncio
 
 
 class command_on_message(BotFunction):
 
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
 
-    async def action(self, message):
-        raise NotImplementedError
+    async def action(self, _):
```

Comment:
Make sure this PR is properly rebased off of master

Reply:
forgot to rebase again